File: CommandLine.c
#include <stdio.h>

#include <stdlib.h>

int main(int argC,char *argU[]){
	int i=0;
	printf("Total number of command %d\n",argC );

	for(int i=0;i<argC; i++){
		printf("%s\n",argU[i]);
	}
	return 0;
}

File: CommandLine.java
import java.util.*;

class CommandLine{
	public static void main(String arg[]){
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter total number of argumnets:" +arg.length);
		System.out.println("Command line argumnets are :");
	
	for(int i=0; i<arg.length; i++){
		System.out.println(arg[i]);
	}
}
	
}